pyproject.toml

```diff
@@ -10,50 +10,49 @@ python = "~3.7"
-# ^ For Django 3 support
+PyMuPDF = "~1.17"
+django-filebrowser = "~3.13"
 django-chunks = { git = "https://github.com/dotKom/django-chunks.git", rev = "e6bf109d4fd286964024c140ac1b8f3d86713540" }
 django-crispy-forms = "^1.9"
 django-extensions = "^2.2"
 django-dynamic-fixture = "^3.1"
 django-recaptcha = "~2.0"           # Google ReCaptcha
-django-oauth-toolkit = "~1.2"       # OAuth2 authentication support
+django-oauth-toolkit = "~1.3"       # OAuth2 authentication support
 django-watson = "~1.5.5"            # Indexed model search lib
 django-reversion = "~3.0"           # Model version history with middleware hooks to all post_save signals
 # django-guardian = "~2.2"            # Per Object permissions framework
 django-guardian = { git = "https://github.com/django-guardian/django-guardian.git", rev = "dba092f5db86808713433388717dcfbaaa928a0b" }
 # ^ TODO: Update when django-guardian > 2.2 is released with fix for `get_objects_for_user` ref. dotkom/onlineweb4#2459
-django-taggit = "~1.2"              # Generic multi-model tagging library
+django-taggit = "~1.3"              # Generic multi-model tagging library
 django-taggit-serializer = "~0.1"   # REST Framework serializers for Django-taggit
 APScheduler = "~3.6"                # Scheduler
 redwine = {git = "https://github.com/dotkom/redwine.git", rev = "1.3.2"}
 reportlab = "~3.5"
-pdfdocument = "~3.3"
+pdfdocument = "~4.0"
 Unidecode = "~1.1"                  # Translates every unicode symbol to the closest ascii. online_mail_usernames
 django-markdown-deux = "~1.0"
-django-cors-headers = "~3.2"        # Enable CORS for Nibble
+django-cors-headers = "~3.3"        # Enable CORS for Nibble
 django-datetime-widget = { git = "https://github.com/oleast/django-datetime-widget.git", rev = "95a5ba4738853d1ed4081e1e33e1600cba984aad" }
-django-webpack-loader = "~0.6"      # Integration with webpack
+django-webpack-loader = "~0.7"      # Integration with webpack
 dj-database-url = "~0.5"            # Allows to configure databases using URLs
-python-decouple = "~3.1"            # Configuration and settings management
+python-decouple = "~3.3"            # Configuration and settings management
 django-oidc-provider = "~0.7"       # OpenID Connect Provider
-docutils = "~0.15"                  # Allows django-admindocs to generate documentation
-requests-oauthlib = "~1.2"          # Simplify OAuth2 connections to external apps
-oic = "~1.0"			            # OpenID Connect Client
+docutils = "~0.16"                  # Allows django-admindocs to generate documentation
+requests-oauthlib = "~1.3"          # Simplify OAuth2 connections to external apps
+oic = "~1.2"			            # OpenID Connect Client
 python-dateutil = "~2.8"            # Simplify date parsing
 coreapi = "~2.3"                    # API Documentation
-requests = { version = "~2.22", extras = [ "security" ] }
+requests = { version = "~2.23", extras = [ "security" ] }
 # Wiki
-wiki = "0.6b1"
+wiki = "0.6b2"
```

Comment:
Have you had a look at the wiki with this version?
Django-wiki upgrades to Bootstrap 4, which might break the styling for parts of it since we use bootstrap 3

Reply:
Oh no, that is quite the breaking change, they even now require `popper.js` for dropdowns :(

Reply:
Yeah, but it was very much about time that they did the upgrade.

I don't think we'll ever upgrade OW4 to Bootstrap 4, so I think we can't ever upgrade this until we either remove the main front-end from OW4, or split the wiki into its own service